src/styles/images/index.md

```diff
@@ -4,57 +4,219 @@ description: Only use images if there’s a real user need
 section: Styles
 backlog_issue_id: 70
 layout: layout-pane.njk
+status: Experimental
+statusMessage: This guidance is currently experimental because <a class="govuk-link" href="#research-on-images">we want to get feedback</a> to validate how useful it is for service teams.
```

Comment:
```suggestion
statusMessage: This guidance is currently experimental because <a class="govuk-link" href="#research-on-images">we want to get feedback</a> to validate how useful it is for service teams.
show_page_nav: true
```

Reply:
Looks good!